TPS_input.py

```diff
@@ -0,0 +1,148 @@
+# Decide whether to run the old code with the full dataset or the new code
+# with selected peptides
+if (FULL_DATASET):
+    # Load excel file of party processed data
+    data_xls = pd.ExcelFile('./data/timeseries/merged_normalized.xlsx')
+    
+    # Create empty data frame for result data
+    result_data = pd.DataFrame()
+    
+    # Load all of the sheets into a list
+    sheet_list = {}
+    index = 0
+    for sheet_name in data_xls.sheet_names:
+        # Load sheet into list
+        sheet_list[index] = data_xls.parse(sheet_name)
+    
+        index += 1
+    
+    
+    # Get rid of all rows except duplicates
+    duplicate_data = sheet_list[0][(sheet_list[0]['peptide'].isin(sheet_list[1]['peptide']))].dropna().reset_index(drop=True)
+    duplicate_data = duplicate_data[(duplicate_data['peptide'].isin(sheet_list[2]['peptide']))].dropna().reset_index(drop=True)
+    
+    # Trim the duplicate data to just the first four rows (information about peptides)
+    result_data = duplicate_data.iloc[:,0:4]
+    
+    # Create variables for the data in A, B, and C
+    data_A = sheet_list[0][(sheet_list[0]['peptide'].isin(duplicate_data['peptide']))].dropna().reset_index(drop=True)
+    data_B = sheet_list[1][(sheet_list[1]['peptide'].isin(duplicate_data['peptide']))].dropna().reset_index(drop=True)
+    data_C = sheet_list[2][(sheet_list[2]['peptide'].isin(duplicate_data['peptide']))].dropna().reset_index(drop=True)
+    
+    # Add the data from sheets A, B, and C respectively
+    result_data = pd.concat([result_data, data_A.iloc[:,4:12]], axis=1, ignore_index=True)
+    result_data = pd.concat([result_data, data_B.iloc[:,4:12]], axis=1, ignore_index=True)
+    result_data = pd.concat([result_data, data_C.iloc[:,4:12]], axis=1, ignore_index=True)
+    
+    print(result_data)
+    
+    # # Get the data for the stats MulitComparison test
+    array_A = np.asarray(data_A.iloc[:,4:12])
+    array_B = np.asarray(data_B.iloc[:,4:12])
+    array_C = np.asarray(data_C.iloc[:,4:12])
+
+
+    
+    # Stack the dataframes into one
+    df = pd.DataFrame()
+    df_A = pd.DataFrame(array_A)
+    df_B = pd.DataFrame(array_B)
+    df_C = pd.DataFrame(array_C)
+    df = pd.concat([df, df_A], axis=1, ignore_index=True)
+    df = pd.concat([df, df_B], axis=1, ignore_index=True)
+    df = pd.concat([df, df_C], axis=1, ignore_index=True)
+    
+    replicates = df.stack().reset_index()
+    replicates = replicates.rename(columns={'level_0': 'id','level_1': 'replicate', 0:'result'})
+    
+    print(replicates)
+
+    result_data.columns = ['peptide', 'protein', 'gene.name', 'modified.sites', '0min', '2min', '4min', '8min', 
+    '16min', '32min', '64min', '128min','0min', '2min', '4min', '8min', 
+    '16min', '32min', '64min', '128min','0min', '2min', '4min', '8min', 
+    '16min', '32min', '64min', '128min']
+
+    # Calculate medians and log2 fold changes
+    medians = pd.DataFrame()
+    log2_fold_changes = pd.DataFrame()
+    for i in result_data.index:
```

Comment:
The first goal should be to get an implementation that works and not worry much about efficiency.  When we go back for optimization, it will be faster to use pandas functional-style operations to apply this transformation to every line of the data frame.